wso2as/templates/5.3.0/repository/conf/user-mgt.xml.erb

```diff
@@ -0,0 +1,365 @@
+<!--
+  ~ Copyright WSO2, Inc. (http://wso2.com)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+        
+<UserManager>
+    <Realm>
+        <Configuration>
+		<AddAdmin>true</AddAdmin>
+                <AdminRole>admin</AdminRole>
+                <AdminUser>
+                     <UserName>admin</UserName>
+                     <Password>admin</Password>
+                </AdminUser>
+            <EveryOneRoleName>everyone</EveryOneRoleName> <!-- By default users in this role sees the registry root -->
+          <%- if @master_datasources['WSO2USER_DB'] -%>
+            <Property name="dataSource"><%= @master_datasources['WSO2USER_DB']['jndi_config'] %></Property>
+          <%- else -%>
+            <Property name="dataSource">jdbc/WSO2CarbonDB</Property>
+          <%- end -%>
```

Comment:
@R-Rajkumar Updated the hiera yaml files with lates structure and removed conditional checks templates.